raw_data/preprocessed/africa/ethiopa_english.txt

```diff
@@ -0,0 +1,82 @@
+PART ONE
+
+Short Title
+(1) This Proclamation may be cited as the "Environmental Impact Assessment Proclamation No. 299/2002." 
+
+Definitions
+(2) In this Proclamation: [1] "Authority" means the Environmental Protection Authority; [2] "Environment" means the totality of all materials whether in their natural state or modified or changed by human; their external spaces and the interactions which affect their quality or quantity and the welfare of human or other living beings, including but not restricted to, land atmosphere, whether and climate, water, living things, sound, odor, taste, social factors, and aesthetics; [3] "Environmental Impact Assessment" means the methodology of identifying and evaluating in advance any effect, be it positive or negative, which results from the implementation of a proposed project or public instrument; [4] "Impact" means any change to the environment or to its component that may affect human health or safety, flora, fauna, soil, air, water, climate, natural or cultural heritage, other physical structure, or in general, subsequently alter environmental, social, economic or cultural conditions; [5] "Licensing Agency" means any organ of government empowered by law to issue an investment permit or a trade or operating license or a work permit or to register a business organization, as the case may be; [6] "Person" means any natural or juridical person; [7] "Pollutant" means any substance whether liquid, solid or gas which directly or indirectly: [a] alters the quality of any part of the receiving environment so as to affect its beneficial use adversely, or [b] produces toxic substances, diseases, objectionable odour, radioactivity, noise, vibration, heat, or any other phenomenon that is hazardous or potentially hazardous to human health or to other living things; [8] "Project" mans any new development activity under any category listed in any directive issued pursuant to this Proclamation, major expansion or alteration or any existing undertaking, or any resumption of work that had been discontinued; [9] "Proponent" means any organ of government if in the public sector or any person if in the private sector that initiates a project; [10] "Public instrument" means a policy, a strategy, a programme, a law or an international agreement; [11] "Region" means any of those parts of Ethiopia specified as regions under Article 47[1] of the Constitution of the Federal Democratic Republic of Ethiopia and, for the purpose of this Proclamation, includes the Addis Ababa and Dire Dawa Administrations; [12] "Regional Environmental Agency" means any regional government organ entrusted by that Region with a responsibility for the protection or regulation of the environment and natural resources.
```

Comment:
Please replace `to, land atmosphere, whether` with `to land, atmosphere, weather`. Also, please replace `"Project" mans` with `"Project" means`.